setup.py

```diff
@@ -1,14 +1,11 @@
-import os
 from setuptools import setup, find_packages
+import subprocess
 
 
 def get_version():
-    basedir = os.path.dirname(__file__)
-    with open(os.path.join(basedir, 'rq_dashboard/version.py')) as f:
-        locals = {}
-        exec(f.read(), locals)
-        return locals['VERSION']
-    raise RuntimeError('No version info found.')
+    """Return the latest git tag from repository in cwd."""
```

Comment:
Ah, you are spot on. I pushed a test build to testpypi the other day and tested it with the following:

```
# Make a temp virtualenv
$ pew-mktmpenv

# The dependencies are not on testpypi so we have to pull them from the main pypi
$ pip instal rq flask times

# Install latest (but not yet released) rq-dashboard
$ pip install rq-dashboard -i https://testpypi.python.org/pypi

# Run it
$ rq-dashboard
RQ Dashboard version 0.3.5
 * Running on http://0.0.0.0:9181/ (Press CTRL+C to quit)
```

But this worked because we are using the wheel, which does not involve `setup.py` (one of it's features in fact). If instead we force use of the other package (which is a sdist) with:

```
$ pip install rq-dashboard -i https://testpypi.python.org/pypi --no-use-wheel
Collecting rq-dashboard
/home/tcorbettclark/.local/share/virtualenvs/9f4fc56de72d5ac/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Downloading https://testpypi.python.org/packages/source/r/rq-dashboard/rq-dashboard-0.3.5.tar.gz (92kB)
    100% |████████████████████████████████| 94kB 2.2MB/s 
    Complete output from command python setup.py egg_info:
    fatal: Not a git repository (or any of the parent directories): .git
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-CBen63/rq-dashboard/setup.py", line 13, in <module>
        version=get_version(),
      File "/tmp/pip-build-CBen63/rq-dashboard/setup.py", line 7, in get_version
        tags = subprocess.check_output(['git', 'tag']).strip().split('\n')
      File "/usr/lib/python2.7/subprocess.py", line 573, in check_output
        raise CalledProcessError(retcode, cmd, output=output)
    subprocess.CalledProcessError: Command '['git', 'tag']' returned non-zero exit status 128
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-CBen63/rq-dashboard
```